README.md

```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
 ```
 
 Now, any command executed with the Docker image will be funneled through ssm-env.
+
+### Usage with Alpine Node Docker Images
```

Comment:
Would you mind making this section a subsection and title it, `Alpine Docker Image`